src/pages/patientView/PatientViewPage.tsx

```diff
@@ -1455,17 +1456,10 @@ export default class PatientViewPage extends React.Component<
                                     key={2}
                                     id={PatientViewPageTabs.ClinicalData}
                                     linkText="Clinical Data"
+                                    className={'patient-clinical-data-tab'}
```

Comment:
Another minor one. We don't need the curly brackets here.
```suggestion
                                    className="patient-clinical-data-tab"
```

Reply:
i could go either way on this.  i don't think it makes a difference